builtin/common/information_formspecs.lua

```diff
@@ -110,6 +119,98 @@ local function build_privs_formspec(name)
 end
 
 
+-- Checks whether the mod with the given name is a separate mod or a part of a modpack
+local function is_mod_sep_or_part_of_mp(modname)
+	local modpath = core.get_modpath(modname)
+	local find_ptn = "[^" .. DIR_DELIM .. "]*.$"
+	local modpart = modpath:match(find_ptn)
+	local mp_part = modpath:sub(1, modpath:len()-modpart:len()-1):match(find_ptn)
+
+	if mp_part ~= "mods" then   -- this is inside of the modpack!
+		return mp_part, modpart
+	else                        -- this is a just separate mod!
+		return modpart
+	end
+end
+
+
+-- Returns a list with the modpack`s mods
+local function get_modpack_mods(mp_name)
+	local modnames = core.get_modnames()
+
+	local modlist = {}
+	for i, mname in ipairs(modnames) do
+		local mpath = core.get_modpath(mname)
+		if mpath:match(DIR_DELIM .. mp_name .. DIR_DELIM) then
```

Comment:
This is very fragile, imagine a modpack named `home` (`/home/` is in `/home/user/.minetest/...`)

Reply:
The check is rewritten in the new commit.